build-tools-integration-tests/pom.xml

```diff
@@ -61,9 +68,27 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.22.0</version>
```

Comment:
should the versions here be defined in properties maybe?

Reply:
This is just an integration test to test that the published poms+jars work. Not really necessary to have dependecy management here.

Reply:
Regarding dependency management in general:
Gradle already has a [more declarative way](https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml), but dependabot hasn't implemented it yet :( https://github.com/dependabot/dependabot-core/issues/3121 .
Having all the dependencies declared in a .toml file would be much nicer than "abusing" the root project as a `platform()` everywhere.